source/developers/platform_example_light.markdown

```diff
@@ -31,7 +31,7 @@ import logging
 import voluptuous as vol
 
 # Import the device class from the component that you want to support
-from homeassistant.components.light import ATTR_BRIGHTNESS, Light, PLATFORM_SCHEMA
+from homeassistant.components.light import ATTR_BRIGHTNESS, Light, PLATFORM_SCHEMA, SUPPORT_BRIGHTNESS
```

Comment:
This was actually already a problem, but can you wrap this line to 79 characters?